kafka_consumer/datadog_checks/kafka_consumer/kafka_consumer.py

```diff
@@ -26,13 +26,16 @@ def __init__(self, name, init_config, instances):
         self.config = KafkaConfig(self.init_config, self.instance)
         self._context_limit = self.config._context_limit
         tls_context = self.get_tls_context()
-        self.client = make_client(self, self.config, tls_context)
+        self.client = make_client(self.config, tls_context, self.log)
 
     def check(self, _):
         """The main entrypoint of the check."""
         # Fetch Kafka consumer offsets
         self.client.reset_offsets()
```

Comment:
This is probably a better discussion in the future, but if we create a client for each check run instead, we won't need to reset the offsets in the clients. It would technically be a change in behavior though, although to the user this shouldn't be a difference.